src/components/Dropdown/DropdownBasic.tsx

```diff
@@ -52,7 +53,49 @@ export const DropdownBasic = (props: IDropdownBasicProps): ReactElement => {
     setVisible(false);
   };
 
-  const containerClasses = clsx('z-50 bg-white', classes.container, {
+  const open = () => {
+    visible || setVisible(true);
+  }
+
+  const close = () => {
+    !visible || setVisible(false);
+  }
+
+  /* keydown on dropdown */
+  const handleKeyDown = (e: KeyboardEvent) => {
+    switch (e.key) {
```

Comment:
As I mention below, we can make this switch statement a little more concise

Reply:
Done